packages/sdk-demo/src/components/EntityIdLoader.tsx

```diff
@@ -88,4 +90,29 @@ export const createEntityUsersMyEntityRequestFn = (
   );
 };
 
+type ErrorBoundaryProps = {
+  children: ReactNode;
+};
+
+class ErrorBoundary extends Component<
+  ErrorBoundaryProps,
+  {
+    isError: boolean;
+  }
+> {
+  constructor(props: ErrorBoundaryProps) {
+    super(props);
+    this.state = { isError: false };
+  }
+
+  static getDerivedStateFromError(error: unknown) {
+    return { isError: true, error: error };
+  }
+
+  render() {
+    if (this.state.isError) return null;
```

Comment:
It’s generally better to provide some fallback UI instead of just returning null

Reply:
Yep, I was thinking about that, but could not figure out what exactly.